pkg/segment/results/mresults/tests/metricsresults_test.go

```diff
@@ -617,3 +617,208 @@ func test_GetResults_Ops(t *testing.T, initialEntries map[uint32]float64, ansMap
 		}
 	}
 }
+
+func Test_GetResults_And(t *testing.T) {
+	results := make(map[string]map[uint32]float64)
+	results["test.metric.1{color:red,type:compact,"] = map[uint32]float64{
```

Comment:
Is `test.metric.1` supposed to have a closing `}`?

Reply:
Actually, in this function the groupId is without `}`. There is somewhere else we will modify the grpID again and add `}` to it.